kuma/users/jinja2/users/user_edit.html

```diff
@@ -51,7 +51,8 @@ <h1 class="user-title">{{ edit_user.username }} <a class="user-title-profile" hr
       <fieldset class="section notitle" id="personal">
         <ul>
           <li id="field-beta" class="field">
-            <label for="{{ user_form.beta.id_for_label }}">{{ user_form.beta.label }}</label> {{ user_form.beta }}
+            {{ user_form.beta }}
```

Comment:
I think this is cleaner on its own line - good change!

Reply:
Glad I could help!